applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -584,6 +622,11 @@ int handle_at_httpc_request(enum at_cmd_type cmd_type)
 		return err;
 	}
 
+	if (httpc.state != HTTPC_INIT) {
+		LOG_ERR("Another request is not finished.");
+		return err;
+	}
+
```

Comment:
double check? "another request" below, too.

Reply:
The deprecated check is removed.